main.go

```diff
@@ -33,12 +33,11 @@ var log = logger.Get()
 var mainLogger = log.WithField("prefix", "MAIN")
 var ProfileFilename *string
 
-// Get our bak end to use, new beack-ends must be registered here
+// Get our bak end to use, new back-ends must be registered here
 func initBackend(profileBackendConfiguration interface{}, identityBackendConfiguration interface{}) {
 
 	AuthConfigStore = &backends.InMemoryBackend{}
-	IdentityKeyStore = &backends.RedisBackend{KeyPrefix: "identity-cache."}
-
+	IdentityKeyStore = &backends.RedisBackend{KeyPrefix: "identity-cache"}
```

Comment:
Not sure if this is typo or intentional change

Reply:
Please, can you elaborate more? I don't understand which typo

Reply:
The change on the KeyPrefix was to maintain an standard across all the other existing prefixes and have only text without dots, semi-colons, etc.